Specs/Number/Japanese/NumberModel.json

```diff
@@ -1,12 +1,12 @@
 [
   {
     "Input": "西九条",
-    "Comment": "PendingValidation",
+    "NotSupported": "dotnet, javascript, python, java",
     "Results": []
   },
   {
     "Input": "一匹",
-    "Comment": "PendingValidation",
+    "NotSupported": "dotnet, javascript, python, java",
```

Comment:
Existing specs that already worked should not be marked as NotSupported in .NET.
Only new spec cases that don't pass (for numbers/units) or any new DateTime spec cases.